test/spec/idx/authenticate.ts

```diff
@@ -108,29 +114,92 @@ describe('idx/authenticate', () => {
 
   describe('error handling', () => {
 
-    it('returns terminal error when invalid username is provided', async () => {
-      const { authClient } = testContext;
-      const errorResponse = IdxErrorAccessDeniedFactory.build();
-      const identifyResponse =  IdentifyResponseFactory.build();
-      identifyResponse.proceed = jest.fn().mockRejectedValue(errorResponse);
-      jest.spyOn(mocked.idx, 'start').mockResolvedValue(identifyResponse);
+    describe('SSR is not enabled', () => {
```

Comment:
Initially I thought `SSR` means Server side rendering, haha

Reply:
ah it should probably be "profile enrollment"